api/lib/config.js

```diff
@@ -73,6 +73,10 @@ module.exports = (function () {
       pixCertif: process.env.DOMAIN_PIX_CERTIF || 'https://certif.pix',
     },
 
+    http: {
+      fetchTimeOut: ms(process.env.FETCH_TIMEOUT || '20s'),
+    },
```

Comment:
Pour donner du contexte
```suggestion
    partner: {
      fetchTimeOut: ms(process.env.FETCH_TIMEOUT || '20s'),
    },
```